downstream/modules/platform/proc-update-aap-container.adoc

```diff
@@ -0,0 +1,51 @@
+[id="proc-update-aap-container"]
+
+= Updating container-based {PlatformNameShort}
+
+Perform a patch update for a container-based installation of Ansible Automation Platform.
+
+.Prerequisites
+
+* Reviewed the release notes for the associated patch release.
+
+* Created a backup of your {PlatformNameShort} deployment. For more information, see xref:proc-backup-aap-container[Backing up controller-based {PlatformNameShort}].
+
+.Procedure
+
+. Download the latest version of the containerized installer from the link:https://access.redhat.com/downloads/content/480/ver=2.5/rhel---9/2.5/x86_64/product-software[{PlatformNameShort} download] page.
```

Comment:
There's an attribute for the download page. It's {PlatformDownloadUrl} so I think you'd format it this way:
link:{PlatformDownloadUrl}[{PlatformNameShort} download]

Reply:
Fixed.